globals.go

```diff
@@ -212,6 +212,10 @@ const (
 	HostAllocatorRoundUp      = "round-up"
 	HostAllocatorRoundDefault = ""
 
+	HostAllocatorWaitsFeedback      = "waits-feedback"
```

Comment:
`HostAllocatorWaitsOverThreshFeedback`?

The long-runtime tasks calculation isn't feedback, it's for prospective planning. It's predictive rather than reactive. So when it comes to feedback, we have either feedback based on waits or none. In the future perhaps we could have some other form of feedback. So

1) It IS somewhat confusing but I'm not really sure what to do about it
2) I am strongly opposed to saying anything about long tasks here as that is not a form of feedback

I could call it QOS rule and include something about long runtime tasks but I feel like that would be just as confusing in a different way.

Reply:
Okay I'm convinced, thanks for the explanation!